jenkins.sh

```diff
@@ -4,6 +4,12 @@
 : "${JENKINS_HOME:="/var/jenkins_home"}"
 : "${COPY_REFERENCE_FILE_LOG:="${JENKINS_HOME}/copy_reference_file.log"}"
 : "${REF:="/usr/share/jenkins/ref"}"
+
+if ! [ -r "${JENKINS_HOME}" -a -w "${JENKINS_HOME}" ]; then
```

Comment:
```suggestion
if ! [ -r "${JENKINS_HOME}" ] && [ -w "${JENKINS_HOME}" ]; then
```

The static analysis with [shellcheck](https://www.shellcheck.net/) fails on this line with the following error:

```text
if ! [ -r "${JENKINS_HOME}" -a -w "${JENKINS_HOME}" ]; then
                                                ^-- SC2166 (warning): Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.
```

=> My suggestion above keeps the same behavior as proposed by @saper earlier in this PR (e.g. no double subshell) while following shellcheck recommendation (less ambiguity due to shell operators)

Reply:
Looks like this introduces a bug... :/